packages/cli/test/example.test.ts

```diff
@@ -0,0 +1,39 @@
+import { prepareEnvironment } from "@gmrchk/cli-testing-library";
+import { CLITestEnvironment } from "@gmrchk/cli-testing-library/lib/types";
+import { join } from "node:path";
+
+let environment: CLITestEnvironment;
+
+beforeAll(async () => {
+  // This will create a sandbox folder under `/var/folders`
+  environment = await prepareEnvironment();
+});
+
+afterAll(async () => {
+  await environment.cleanup();
+});
+
+describe('cli', () => {
+  // can be any path with a nextjs project
+  const NEXT_PROJECT_PATH = join(__dirname, '..', '..', '..', 'examples', 'nextjs-example');
+
+  it('should be able to execute cli', async () => {
+    const { waitForText, getStdout, wait, pressKey } = await environment.spawn('node', `${join(__dirname, '..', 'dist', 'index.js')} init -p ${NEXT_PROJECT_PATH}`)
```

Comment:
In this case, I'm unsure if we should use `node` and execute from `dist/index.js` or run directly from the typescript file.

I noticed that we use `tsup`, so if we use `node` to execute, probably we'll need to `pnpm dev` and then `pnpm test`

Reply:
This is the correct approach. 

You just need to add a dependency in the [turbo.json](https://github.com/triggerdotdev/trigger.dev/blob/12de1b0d34911e568505f417d0dfb8831ccbc6d4/turbo.json#L49) file. This should look like this:

```json
"test": {
    "dependsOn": ["build"],
    "outputs": []
  },
```

Reply:
That will ensure that it is built first. It will also rebuild automatically if a file has changed, and cache appropriately.